README.md

```diff
@@ -5,8 +5,25 @@ ZGrab is a fast, modular application-layer network scanner designed for completi
 
 ZGrab 2.0 contains a new, modular ZGrab framework, which fully supersedes https://github.com/zmap/zgrab.
 
+## Installation
 
-## Building
+### With Docker
+
+You can run ZGrab 2.0 with our official Docker image. For example, to scan a single website using the HTTP module, you can use:
+
+```shell
+echo 'example.com' | docker run --rm -i ghcr.io/zmap/zgrab2 http
+```
+
+For more complex scanning scenarios, such as using multiple modules or custom configurations, you can create a configuration file and pass it to the container:
+
+```shell
+docker run --rm -i -v /path/to/your/config.ini:/config.ini ghcr.io/zmap/zgrab2 multiple -c /config.ini
+```
+
+Replace `/path/to/your/config.ini` with the path to your configuration file on the host machine. See follow for more details on configurations.
```

Comment:
Can we link to the section on Multiple Module Usage?
Something like
"See Multiple Module Usage for more details on configurations."

Where "Multiple Module Usage" is a link to that section of the README

Reply:
8d2bf0422fe1dd9fc2639270da2c78c7ec9365de